### ISSUE TITLE
data `volterra_http_loadbalancer_state` does not show DNS records for all domains (where UI does)

### ISSUE THREAD

**Issue by andyt10** [volterraedge/terraform-provider-volterra#214]
Upon adding a secondary entry to the `domains` list of a `volterra_http_loadbalancer` I noticed that the DNS validation records are not shown in an accompanying `volterra_http_loadbalancer_state` data source.

Removing and re-adding the data source object doesn't even seem to prompt a refresh (to show both required values), despite the fact the F5 Console shows validation records required for both domains.

This inhibits the ability to automatically validate domain ownership for certificate reasons.

**Comment by andyt10** [comment]
Just to give some extra context, here's a (heavily) redacted `terraform apply` result showing the outputs even after replacing the data object

I would expect to see two entries in the `dns_records` list

```  
+ testoutput = {
      + auto_cert_info  = [
          + {
              + auto_cert_expiry  = "nil"
              + auto_cert_issuer  = ""
              + auto_cert_state   = 12
              + auto_cert_subject = ""
              + dns_records       = [
                  + {
                      + name  = "<REDACTED>"
                      + type  = "CNAME"
                      + value = "<REDACTED>"
                    },
                ]
            },
        ]
      + auto_cert_state = "PreDomainChallengePending"
      + cname           = "<REDACTED>.ves.io"
      + id              = "<REDACTED>"
      + ip_address      = "<REDACTED>"
      + name            = "<REDACTED>"
      + namespace       = "non-prod"
      + state           = "VIRTUAL_HOST_PENDING_A_RECORD"
   ```

**Comment by andyt10** [comment]
If I query the API directly for the same resource the `dns_records` section does show multiple values.

There appears to be a part of the provider code where it 'flattens' down the records and expects only 1 entry though I'm not quite sure why. I believe this is the issue.

https://github.com/volterraedge/terraform-provider-volterra/blob/main/volterra/data_source_http_loadbalancer_state.go#L163

**Comment by awasilyev** [comment]
that was solved in the version 0.11.28 of provider
